modular_skyrat/modules/Kinetic_destroyer/code/modules/projectiles/guns/energy/kinetic_accelerator.dm

```diff
@@ -0,0 +1,279 @@
+//premium accel
+/obj/item/gun/energy/kinetic_accelerator/premiumka
+	icon = 'modular_skyrat/modules/Kinetic_destroyer/code/modules/mining/icons/obj/guns/energy.dmi'
+	name = "Premium accelerator"
+	desc = "A premium kinetic accelerator fitted with an extended barrel and increased pressure tank."
+	icon_state = "kineticgun"
+	inhand_icon_state = "kineticgun"
+	lefthand_file = 'modular_skyrat/modules/Kinetic_destroyer/code/modules/mining/icons/mobs/inhands/weapons/guns_lefthand.dmi'
+	righthand_file = 'modular_skyrat/modules/Kinetic_destroyer/code/modules/mining/icons/mobs/inhands/weapons/guns_righthand.dmi'
+	ammo_type = list(/obj/item/ammo_casing/energy/kinetic/premiumka)
+	cell_type = /obj/item/stock_parts/cell/emproof
+	item_flags = NONE
+	obj_flags = UNIQUE_RENAME
+	weapon_weight = WEAPON_LIGHT
+	can_flashlight = TRUE
+	flight_x_offset = 15
+	flight_y_offset = 9
+	automatic_charge_overlays = FALSE
+	can_bayonet = TRUE
+	knife_x_offset = 20
+	knife_y_offset = 12
+	overheat_time = 16
+	holds_charge = FALSE
+	unique_frequency = FALSE // modified by KA modkits
+	overheat = FALSE
+	max_mod_capacity = 100
+	recharge_timerid
```

Comment:
every single one of these is filled with unnecessary shitty copypaste from the parent type